.github/workflows/main.yaml

```diff
@@ -419,6 +469,15 @@ jobs:
           asset_path: artifacts/wasmer-linux-aarch64/wasmer.tar.gz
           asset_name: wasmer-linux-aarch64.tar.gz
           asset_content_type: application/gzip
+      - name: Upload Release Asset Linux (musl) amd64
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: artifacts/wasmer-linux-musl-amd64/wasmer.tar.gz
+          asset_name: wasmer-linux-musl-amd64.tar.gz
+          asset_content_type: application/gzip
```

Comment:
This is the duplicated code I'm mentionning in my comment above (line 435).

We should keep this block, and remove the previous one. The `id` key isn't necessary.